### ISSUE TITLE
Cleanup README.md

### ISSUE THREAD

**Issue by pganguli** [CodeFlow201/forestTreesTagging#1]
In its current state, the README.md contains multiple typos and grammar errors.
It is proposed that these errors be fixed, so that the visibility of this repo increases.

**Comment by pganguli** [comment]
@vaibhavarora102 Can this issue kindly be assigned to me for GSSOC'21 ?

**Comment by ghackg11** [comment]
The contribution phase has not been started yet. It's from the 8th of March, you can contribute only after that.  Moreover, you can add some interesting stuff to the readme (which must be relevant too).

**Comment by pganguli** [comment]
> The contribution phase has not been started yet. It's from the 8th of March, you can contribute only after that. Moreover, you can add some interesting stuff to the readme (which must be relevant too).

Thanks for the heads up; I was not aware of the updated schedule. I'm closing my current PR to work on it a bit more, so that I can submit another PR after the 8th of March.

**Comment by ThatOneTallKid** [comment]
Can this issue be assigned to me ?

**Comment by ghackg11** [comment]
There is no such thing as issue assigning. The candidates who give PR first resolving an already made issue, or make a new issue and solve it will be considered. All this is based on a first come first serve basis. @ThatOneTallKid

**Comment by ghackg11** [comment]
> Can this issue be assigned to me?

I am not assigning this to anyone. Any candidate who gives PR first and does relevant editing and additions will have his or her PR accepted. You can just assign any issue to yourself.
